From 7cf7ac4eb2c29a2b1aa748dae7658da5fbbc3a18 Mon Sep 17 00:00:00 2001
From: Brian Parsons <brian@pmex.com>
Date: Mon, 11 Jun 2012 17:30:10 -0400
Subject: [PATCH 1/2] Update module filter to be aware of hyphens in the
 MODULES array

[dave: fix whitespace and re-add needed quoting]

Signed-off-by: Dave Reisner <dreisner@archlinux.org>
---
 functions |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/functions b/functions
index 4a62d8e..e9fb81a 100644
--- a/functions
+++ b/functions
@@ -590,7 +590,7 @@ write_image_config() {
         . "$CONFIG"
 
         # sanitize of any extra whitespace
-        read -ra modules <<< "$MODULES"
+        read -ra modules <<<"${MODULES//-/_}"
         for mod in "${modules[@]}"; do
             in_array "${mod%\?}" "${ADDED_MODULES[@]}" || continue
             add+=("${mod%\?}")
-- 
1.7.10.4

